src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -734,8 +734,8 @@ private static bool NeedToClearProcessModulePath(string currentProcessModulePath
                 // so if the current process module path contains any of them, it's likely that the sxs
                 // ps was started directly on windows, or from full ps. The same goes for the legacy personal
                 // and shared module paths.
-                string hklmModulePath = GetExpandedEnvironmentVariable("PSMODULEPATH", EnvironmentVariableTarget.Machine);
-                string hkcuModulePath = GetExpandedEnvironmentVariable("PSMODULEPATH", EnvironmentVariableTarget.User);
+                string hklmModulePath = GetExpandedEnvironmentVariable("PSModulePath", EnvironmentVariableTarget.Machine);
```

Comment:
I count 8 references to this same string in code - there should be a `const string` that we use instead.